package.json

```diff
@@ -67,11 +67,11 @@
   "license": "MIT",
   "homepage": "https://www.elastic.co/guide/en/apm/agent/rum-js/current/index.html",
   "devDependencies": {
-    "@angular-devkit/build-angular": "^12.0.4",
-    "@angular/cli": "^12.0.4",
+    "@angular-devkit/build-angular": "^13.1.2",
```

Comment:
we have to test angular 13 as we have separate angular package for RUM. Lets not upgrade frameworks(react, angular) as part of this deps upgrade PR.